utils/upload_ansible_roles_to_galaxy.py

```diff
@@ -381,6 +383,7 @@ def main():
     # Create empty repositories
     github_new_repos = sorted(list(set(selected_roles) - set(github_repositories)))
     if github_new_repos:
+        repo_status = "new"
```

Comment:
If there is a single new repo, won't all repos be update as `new`, and possibly have their readme and content updated?

Reply:
That's a good point @yuumasato I didn't think about that. Thanks! I will look into it.

Reply:
@yuumasato fixed.